server/ai_http.go

```diff
@@ -125,7 +143,31 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 		controlPubCh := trickle.NewLocalPublisher(h.trickleSrv, mid+"-control", "application/json")
 		controlPubCh.CreateChannel()
 
-		// Subscribe to the publishUrl for payments monitoring
+		paymentReceiver := livePaymentReceiver{orchestrator: h.orchestrator}
+		priceInfo, err := h.orchestrator.PriceInfo(sender, core.ManifestID(mid))
+		if err != nil {
+			respondWithError(w, err.Error(), http.StatusInternalServerError)
+			return
+		}
+		f := func(inPixels int64) error {
+			err := paymentReceiver.AccountPayment(context.Background(), &SegmentInfoReceiver{
+				sender:    sender,
+				inPixels:  inPixels,
+				priceInfo: priceInfo,
+				sessionID: mid,
+			})
+			if err != nil {
+				slog.Warn("Error accounting payment, stopping stream processing", "err", err)
+				// TODO: Check if we do not need to any other cleanup
+				pubCh.Close()
```

Comment:
@j0sh is it how we plan to stop the stream (by closing the streams) and also tear down the container (work related to what @victorges is doing)?